docs/topics/mta-requirements-for-java-analyzer.adoc

```diff
@@ -6,11 +6,11 @@
 [id="mta-requirements-for-java-analyzer_{context}"]
 = Adding minimum requirements for Java analyzer and discovery
 
-There is a minimum requirement for the Java analyzer, and also the discovery task, which by default is set to 2 GB.
+The default minimum requirement for the Java analyzer, and also the discovery task, is set to 2.5 GB.
```

Comment:
I prefer the previous phrasing that emphasizes this is a minimum value (and users should not set it to lower than that)